src/index.ts

```diff
@@ -41,18 +41,21 @@ const getTrayIcon = () => {
 const windowDimsSettingsKey = 'overlayed.window'
 
 const allocMainWindow = () => {
+  
+  // create our window
+  mainWindow = new CompositeWindow()
 
   // see if we have window settings, if so, use them
-  const windowDims = settings.get(windowDimsSettingsKey, {
-    height: 480,
-    width: 680,
-    x: 0,
-    y: 0,
-  }) as {x: number, y: number, width: number, height: number}
+  // if there is no window settings available 
+  // we read the size of primary display and then adjusting the window size to it
```

Comment:
nitpick comment: grammar should be "and then adjust the window size to match it"

Reply:
English is not my first language 😃 feel free to nitpick about my grammar 😄 it only makes it better